pgcrypto/functions.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.db.models import Func
+
+from .base import aes_pad_key
+
+
+class Encrypt(Func):
+    function = 'encrypt'
+    template = "armor(%(function)s(convert_to(nullif(%(expressions)s, ''), %%s), %%s, %%s))"
+
+    DEFAULT_CIPHER = getattr(settings, "PGCRYPTO_DEFAULT_CIPHER", "aes")
+    DEFAULT_KEY = getattr(settings, "PGCRYPTO_DEFAULT_KEY", settings.SECRET_KEY)
```

Comment:
It's minor, but I think these `getattr`s should happen inside the `__init__` instead of at class definition time. I know you shouldn't change Django settings "live" but I've seen it done.